pkgs/dart_mcp/example/dash_client.dart

```diff
@@ -35,22 +35,23 @@ final argParser =
       help: 'A command to run to start an MCP server',
     );
 
-final class GeminiClient extends MCPClient with RootsSupport {
+final class DashClient extends MCPClient with RootsSupport {
   final StreamQueue<String> stdinQueue;
   final List<String> serverCommands;
   final List<ServerConnection> serverConnections = [];
-  final List<gemini.Tool> serverTools = [];
   final Map<String, ServerConnection> connectionForFunction = {};
-  final gemini.GenerativeModel model;
   final List<gemini.Content> chatHistory = [];
+  final gemini.GenerativeModel model;
 
-  GeminiClient(this.serverCommands, {required String geminiApiKey})
-    : stdinQueue = StreamQueue(
-        stdin.transform(utf8.decoder).transform(const LineSplitter()),
-      ),
-      model = gemini.GenerativeModel(
-        model: 'gemini-2.5-pro-exp-03-25',
+  DashClient(this.serverCommands, {required String geminiApiKey})
+    : model = gemini.GenerativeModel(
+        // model: 'gemini-2.5-pro-exp-03-25',
+        model: 'gemini-2.0-flash',
```

Comment:
which model do we want to use? should we delete the commented out one?

Reply:
Its handy to be able to swap back and forth.... flash is a lot faster so I find it better for this purpose. But its not as good at analyzing images for instance.

Reply:
Going to go ahead and land this as is - we should probably have a followup which just allows you to select a model on the fly.